docs/content/testing/writing-a-new-test.md

```diff
@@ -0,0 +1,340 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Writing a new test
+
+This document walks through creating tests for a made-up class to explain how
+ the parts of the Unity GDK can be tested.
+
+## Prerequisites
+
+Please read through and understand the [Testing Overview](./testing-overview.md)
+ and the [Testing Guidelines](./testing-guidelines.md) documents first.
+
+## Using NUnit
+
+To help get started with testing, we recommend these documents which help cover
+ the basics:
+
+- [Introduction to NUnit](https://www.youtube.com/watch?v=1TPZetHaZ-A):
+  - Skip the part between 0:40 - 2:49
```

Comment:
Do you mean "skip *to* the part between"? (and below)

Reply:
No, I was trying to say these parts are not relevant, I'll say it directly